mllib/src/main/scala/org/apache/spark/ml/optim/loss/RDDLossFunction.scala

```diff
@@ -50,7 +50,7 @@ private[ml] class RDDLossFunction[
     Agg <: DifferentiableLossAggregator[T, Agg]: ClassTag](
     instances: RDD[T],
     getAggregator: (Broadcast[Vector] => Agg),
-    regularization: Option[DifferentiableRegularization[Array[Double]]],
+    regularization: Option[DifferentiableRegularization[Vector]],
```

Comment:
Sure. But it seems a little strange to make this a Spark `Vector` while `RDDLossFunction` is for a Breeze `Vector`.